tests/icon-api.js

```diff
@@ -0,0 +1,129 @@
+import test from 'brittle'
+import RAM from 'random-access-memory'
+import Database from 'better-sqlite3'
+import { drizzle } from 'drizzle-orm/better-sqlite3'
+import { migrate } from 'drizzle-orm/better-sqlite3/migrator'
+
+import IconApi from '../src/icon-api.js'
+import { DataType } from '../src/datatype/index.js'
+import { DataStore } from '../src/datastore/index.js'
+import { createCoreManager } from './helpers/core-manager.js'
+import { iconTable } from '../src/schema/project.js'
+import { IndexWriter } from '../src/index-writer/index.js'
+
+const smallIcon = 'myIconSmall'
```

Comment:
the blobs representing an actual icon are simple strings. Does it make sense to test with an actual icon - that we commit to the repo or load from a url?? -

Reply:
It's fine, but can be a bit confusing reading these tests. Why not just test with random buffers, e.g. `smallIcon = randomBytes(100)`

Reply:
Best to get as close to real use as possible